src/components/FilterPanel/FilterPanel.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import { FilterPanelContainer, FilterButton } from './FilterPanel.styles';
+
+import type { FilterPanelProps } from './FilterPanel.types';
+
+export const FilterPanel: React.FC<FilterPanelProps> = ({
+  filters,
+  selectedValue,
+  onSelect,
+  panelVariant = 'primary',
+}) => {
+  return (
+    <FilterPanelContainer
+      role="radiogroup"
+      aria-label="Test filters"
+      data-testid="filter-panel"
+      panelVariant={panelVariant}
+    >
+      {filters.map((filter) => {
+        const isSelected = selectedValue === filter.value;
+        return (
+          <FilterButton
+            key={filter.value}
+            selected={isSelected}
+            panelVariant={panelVariant}
+            size="medium"
+            role="radio"
+            aria-checked={isSelected}
+            data-testid={`filter-${filter.value}`}
+            onClick={() => onSelect(filter.value)}
+          >
+            {filter.label}
+          </FilterButton>
+        );
+      })}
+    </FilterPanelContainer>
+  );
+};
+
+export default FilterPanel;
```

Comment:
The file ends with a default export: `export default FilterPanel;`. The project rules explicitly require named exports (NO default exports). This also causes inconsistency with CDD conventions and stories.

**Sugerencia:**
```typescript
Remove the default export and import the component via named imports in other files. Example: delete line `export default FilterPanel;` and ensure stories change to `import { FilterPanel } from './FilterPanel';`
```